Assets/HoloToolkit/ARCA/Scripts/Recording/ReplayKitRecorder.cs

```diff
@@ -131,9 +222,12 @@ public void StartRecording()
             #endif
         }
 
+        /// <summary>
+        /// Stops the recording process
+        /// </summary>
         public void StopRecording()
-		{
-            #if PLATFORM_IOS
+        {
+#if PLATFORM_IOS
```

Comment:
Is this a custom define?

I'm not really sure we should define our own. [Unity should have definitions we can use](https://docs.unity3d.com/Manual/PlatformDependentCompilation.html).
use `UNITY_IOS`